libraries/Adapters/Microsoft.Bot.Builder.Adapters.Slack/SlackAdapter.cs

```diff
@@ -193,19 +193,22 @@ public override async Task<ResourceResponse[]> SendActivitiesAsync(ITurnContext
         /// <returns>A resource response with the Id of the updated activity.</returns>
         public override async Task<ResourceResponse> UpdateActivityAsync(ITurnContext turnContext, Activity activity, CancellationToken cancellationToken)
         {
-            if (activity.Id != null && activity.Conversation != null)
+            if (activity.Id == null)
```

Comment:
Can we add a prev validation for activity == null? Same on turnContext

Reply:
Done. Thanks!